ansible/configs/ocp4-workshop/destroy_env.yml

```diff
@@ -389,4 +390,4 @@
       when:
         # delete stack if stack creation failed
         - stack_status is defined
-        - stack_status != 'CREATE_COMPLETE'
+        - stack_status != 'CREATE_COMPLETE' or stack_status != 'UPDATE_COMPLETE'
```

Comment:
this should probably be an `and`?

Reply:
Agreed, dropped into another list item to make `and` as this makes more sense.